arex-web-core/src/main/java/com/arextest/web/core/business/SchemaInferService.java

```diff
@@ -187,8 +186,6 @@ public SyncResponseContractResponseType syncResponseContract(SyncResponseContrac
         List<AppContractDto> upserts = new ArrayList<>();
         AppContractDto entryPointApplication = new AppContractDto();
         entryPointApplication.setOperationId(operationId);
-        entryPointApplication.setOperationName(latestEntryCompareResult.getOperationName());
```

Comment:
entryPoint don't need operationType & operationName? why?

Reply:
EntryPoint's operationName & operationType could be saved but not used for query.